test/Microsoft.AspNetCore.Razor.Evolution.Test/DefaultDirectiveIRPassTest.cs

```diff
@@ -1,171 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
I've resurrected this guy here https://github.com/aspnet/Razor/pull/1071/commits